angrmanagement/config/config_manager.py

```diff
@@ -1,7 +1,31 @@
 
-from PySide2.QtGui import QFont, QFontMetricsF
+from PySide2.QtGui import QFont, QFontMetricsF, QColor
 
 from .config_entry import ConfigurationEntry as CE
+import toml
+import logging
```

Comment:
Same - move them to the top of the file.

In general, the imports should follow the following order:

```
Modules come with Python

Third-party modules

angr modules

Relative imports from the current package
```